src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -115,6 +104,12 @@ impl KvmVcpu {
         self.mpidr =
             arch::aarch64::regs::read_mpidr(&self.fd).map_err(Error::ConfigureRegisters)?;
 
+        if vcpu_config.cpu_template == CpuFeaturesTemplate::V1N1 {
```

Comment:
line 107: `V1N1`, line 108: `n1v1`. Let's choose one of them.

Reply:
v1n1 it is